datasets/varen/varen.json

```diff
@@ -0,0 +1,175 @@
+  },
+  "accrualPeriodicity": "maandelijks",
+  "spatialDescription": "Gemeente Amsterdam",
+  "theme": "Verkeer",
+  "publisher": "OIS",
+  "owner": "Gemente Amsterdam, Toezicht en Handhaving Openbare Ruimte",
+  "authorizationGrantor": "Toezicht en Handhaving Openbare Ruimte",
+  "keywords": "ligplaatsenregister, varen, opstaplocatie, afstaplocatie, grachten, varen",
+  "hasBeginning": "2020-01-01T00:00:00",
+  "crs": "EPSG:28992",
+  "tables": [
+    {
+      "id": "var_opafstapplaats",
+      "type": "table",
+      "provenance": "var_opafstapplaats",
+      "title": "var_opafstapplaats",
+      "description": "Deze dataset bevat de gegevens over op- en afstaplocaties voor pleziervaart, passagiersvaart en transport over het water. De gemeente wil daarbij voldoende ruimte voor bewoners creëren, onder andere doorbeheersing van het aantal vaarbewegingen en het spreiden van plezier- en passagiersvaart.",
+      "schema": {
+        "required": ["volgnr"],
+        "mainGeometry": "geometrie",
+        "display": "volgnr",
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
+          "id": {
+            "type":"string",
+            "provenance": "var_opafstapplaats_id",
+            "description": "Uniek identificerend kenmerk van het record"  
+          },
+          "schema": {
+            "$ref": "https://schemas.data.amsterdam.nl/schema@1.1.1#/definitions/schema"
+          },
+          "volgnr": {
+            "type": "string",
+            "description": "Volgnummer van het object"
+          },
+          "tekstOnMouseover": {
+            "type": "string",
+            "provenance": "tekst_on_mouseover",
+            "description": "Volgnummer + locatiebeschrijving"
+          },
+          "kleurOpKaart": {
+            "type": "string",
+            "provenance": "kleur_op_kaart",
+            "description": "Gewenste kleurweergave op de kaart"
+          },
+          "opEnAfstap": {
+            "type": "string",
+            "provenance": "op_en_afstap",
+            "description": " Indicatie ove opstap of afstaplocatie"
+          },
+          "LaadLos": {
+            "type": "string",
+            "provenance": "laad_los",
+            "description": " Indicatie voor laad en los doeleinden."
+          },
+          "eLaadpunt": {
+            "type": "string",
+            "provenance": "e_laadpunt",
+            "description": "Indicatie electrische laadpunt voorziening."
+          },
+          "geometrie": {
+            "$ref": "https://geojson.org/schema/Geometry.json",
+            "description": "Geometrie in RD (epsg:28992)"
+          },
+          "verwerkingsdatum": {
+            "type": "string",
+            "format": "date-time",
+            "provenance": "verwerkingsdatum",
+            "description": "Datum waarop het object  in het systeem is geregistreerd"
+          },
+          "gbdBuurtId": {
```

Comment:
Als een veld een relatie is (foreign key is) naar een andere tabel, dan hoef je in de naam geen "Id" worden toegevoegd. Dit gebeurt dan onderwater automatisch. Dus in dit geval zou de naam worden: "gbdBuurt" bijvoorbeeld. Je kunt er ook meer een uitgeschreven naam van maken bijvoorbeeld: "ligtInBuurt"